cmb2-conditionals.php

```diff
@@ -83,13 +83,10 @@ class CMB2_Conditionals {
 		/**
 		 * Constructor - Set up the actions for the plugin.
 		 */
-		public function __construct() {
-			if ( ! defined( 'CMB2_LOADED' ) || false === CMB2_LOADED ) {
-				return;
-			}
+		public function __construct() {			
 
-			add_action( 'admin_init', array( $this, 'admin_init' ), self::PRIORITY );
-			add_action( 'admin_footer', array( $this, 'admin_footer' ), self::PRIORITY );
+			add_action( 'cmb2_admin_init', array( $this, 'admin_init' ), self::PRIORITY );
+			add_action( 'cmb2_footer_enqueue', array( $this, 'admin_footer' ), self::PRIORITY );
 
```

Comment:
If I remember correctly (but I could well be wrong), these hooks weren't always available in CMB2.
So, since which version of CMB2 have they been available ? And does that mean there is a change in the minimum CMB2 requirement for this plugin if this change would be applied ?
If so, this will need to be annotated somewhere very clearly or else, this should become an `if/else` based on checking a CMB2 version constant to see if these hooks are available and if not, fall back to the old code.